### ISSUE TITLE
The command language set in the settings does not work in the settings.

### ISSUE THREAD

**Issue by kberkut** [LibreCAD/LibreCAD#1602]
The command language set in the settings does not work in the settings

![image](https://user-images.githubusercontent.com/57869227/231943148-39ed906c-3ac9-4dd1-92cf-46c181414bbe.png)

### Expected behavior
Work command on english

### Observed behavior
English command not work.

![image](https://user-images.githubusercontent.com/57869227/231943314-60c3aa09-6775-46cb-a8a2-0c24c7e1659f.png)

### Steps to reproduce or sample file
Set the interface language to Russian and the language for entering commands is English.

### Operating System
OS: Arch Linux x86_64 
Kernel: 6.2.10-arch1-1 
Resolution: 1920x1080 
DE: GNOME 43.4 
CPU: AMD Ryzen 5 3500U with Radeon Vega Mobile Gfx (8) @ 2.100GHz 
GPU: AMD ATI Radeon Vega Series / Radeon Vega Mobile Series 
Memory: 5635MiB / 9886MiB  

### LibreCAD version info
Версия: 2.2.0
Компилятор: GNU GCC 9.4.0
Собран: Dec 17 2022
Версия Qt: 5.12.9
Версия Boost: 1.75.0
Система: Arch Linux

**Comment by otula71** [comment]
> 1, English comands and short commands always work; 

That's not true. I have the same problem as kberkut. I don't use Russian, but Czech. I also tried Slovak. Same problem everywhere. If English isn't set for the GUI, then the English commands don't work.

![librecad](https://github.com/LibreCAD/LibreCAD/assets/49908041/6224e24a-1966-4a17-9d7f-75cf5e42c11e)
![librecad2](https://github.com/LibreCAD/LibreCAD/assets/49908041/6ade0f21-a149-4cde-8586-d11b0266dd2a)
